exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>This is the first time it's mentioned that the stars we were working with were in the SMC, so that can go in the introduction.</p><p> </p><p>Also, it may be leftover language from before this tutorial was split into 2 parts, but we haven't really obtained the extinction curve yet, right? So maybe "Try to obtain the spectra necessary to create an extinction curve..." etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>